handbook/engineering/README.md

```diff
@@ -751,6 +751,27 @@ When this occurs, we will begin receiving the following error message when attem
 
 5. Accept the new terms of service.
 
+### Managing MDM CSR Certificate
+
+This certificate is currently set to expire Oct 2, 2024 and needs to be renewed prior to this. This is notified to the team by the MDM calendar event [IMPORTANT: Renew MDM CSR Certificate](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NHM3YzZja2FoZTA4bm9jZTE3NWFvMTduMTlfMjAyNDA5MTFUMTczMDAwWiBjXzI0YjkwMjZiZmIzZDVkMDk1ZGQzNzA2ZTEzMWQ3ZjE2YTJmNDhjN2E1MDU1NDcxNzA1NjlmMDc4ODNiNmU3MzJAZw&tmsrc=c_24b9026bfb3d5d095dd3706e131d7f16a2f48c7a505547170569f07883b6e732%40group.calendar.google.com&scp=ALL)
+
+Steps to renew the certificate:
+
+1. Visit the [Apple developer account login page](https://appleid.apple.com/account?appId=632&returnUrl=https%3A%2F%2Fdeveloper.apple.com%2Fcontact%2F).
+2. Log in using the credentials stored in 1Password under "Apple developer account".
+3. Verify you are using the "Enterprise" subaccount for Fleet Device Management Inc.
+4. Generate a new certificate following the instructions in [MicroMDM](https://github.com/micromdm/micromdm/blob/c7e70b94d0cfc7710e5c92be20d4534d9d5a0640/docs/user-guide/quickstart.md?plain=1#L103-L118).
+5. Note: `mdmctl` will generate a `VendorPrivateKey.key` and `VendorCertificateRequest.csr` using `billing@fleetdm.com` and a passphrase (suggested generation method with pwgen available in brew / apt / yum `pwgen -s 32 -1vcy`)
```

Comment:
Including the full @fleetdm.com email seems to set off a website linting error (IIRC we try to avoid putting full emails because it increases spam) - maybe just use `billing@`?

Reply:
changed